components/proxy/src/main/java/com/hotels/styx/metrics/reporting/graphite/GraphiteReporterService.java

```diff
@@ -63,66 +56,53 @@ private GraphiteReporterService(Builder builder) {
     }
 
     @Override
-    public CompletableFuture<Void> start() {
+    protected CompletableFuture<Void> startService() {
         return CompletableFuture.runAsync(() -> {
-            LOGGER.info("Graphite started on address=\"{}:{}\"", host, port);
             this.reporter.start(reportingIntervalMillis, MILLISECONDS);
+            LOGGER.info("Graphite service started name=\"{}\"", name);
```

Comment:
Maybe `LOGGER.info("Graphite service started: serviceName=\"{}\"", name);`

Reply:
I disagree. These messages are for our end users. They are not interested in seeing styx variable names, or log messages in the camel-case notation.

Reply:
I was more concerned with potential confusion when reading. 
It seems like the word "name" just suddenly appears, with no connection to the previous clause. Separating it with some kind of punctuation mark might do a lot to improve this.

Reply:
I see. Makes sense. I will change this.